### PR TITLE
Add stub navigation for the project, profile and listings

### PR DESCRIPTION
**Pull request by dominika-kokowicz**

### CODE REVIEW

File: App/HejtoBuzz/HejtoBuzz/Views/Dashboard/ListingView.swift [hunk 1]
```diff
@@ -0,0 +1,59 @@
+//
```
Comment (sloik) [inline App/HejtoBuzz/HejtoBuzz/Views/Dashboard/ListingView.swift:1]: Please remove the header comment :)

File: App/HejtoBuzz/HejtoBuzz/Views/Dashboard/MainDashboardView.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
+//
```
Comment (sloik) [inline App/HejtoBuzz/HejtoBuzz/Views/Dashboard/MainDashboardView.swift:1]: Please remove the header comment :)

File: App/HejtoBuzz/HejtoBuzz/Views/Dashboard/ProfileView.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
+//
```
Comment (sloik) [inline App/HejtoBuzz/HejtoBuzz/Views/Dashboard/ProfileView.swift:1]: Please remove the header comment :)

File: App/HejtoBuzz/HejtoBuzz/Views/Theme.swift [hunk 4]
```diff
@@ -0,0 +1,12 @@
+//
```
Comment (sloik) [inline App/HejtoBuzz/HejtoBuzz/Views/Theme.swift:1]: Remove the comment ;)

**Review by sloik — CHANGES_REQUESTED** [review]
Good job :)

File: App/HejtoBuzz/HejtoBuzz.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -340,10 +360,14 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				69F8C42E29C5F484002DD6A7 /* MainDashboardView.swift in Sources */,
```
[the remark this answers is not in the document]
Comment (dominika-kokowicz) [reply App/HejtoBuzz/HejtoBuzz.xcodeproj/project.pbxproj:363]: Now yes - fixed :D

File: App/HejtoBuzz/HejtoBuzz/Views/Dashboard/ListingView.swift (hunk 1 above)
Comment (dominika-kokowicz) [reply App/HejtoBuzz/HejtoBuzz/Views/Dashboard/ListingView.swift]: Fixed :)

File: App/HejtoBuzz/HejtoBuzz/Views/Dashboard/MainDashboardView.swift (hunk 2 above)
Comment (dominika-kokowicz) [reply App/HejtoBuzz/HejtoBuzz/Views/Dashboard/MainDashboardView.swift]: Fixed :)

File: App/HejtoBuzz/HejtoBuzz/Views/Theme.swift (hunk 4 above)
Comment (dominika-kokowicz) [reply App/HejtoBuzz/HejtoBuzz/Views/Theme.swift]: Fixed :)

**Pull request merged**